api/src/models/pitch.ts

```diff
@@ -28,8 +28,18 @@ const Pitch = new mongoose.Schema({
     default: assignmentStatusEnum.NONE,
   },
   assignmentGoogleDocLink: { type: String, default: null },
-  assignmentContributors: [{ type: Schema.Types.ObjectId, ref: 'User' }],
-  pendingContributors: [{ type: Schema.Types.ObjectId, ref: 'User' }],
+  assignmentContributors: [
+    {
+      userId: { type: Schema.Types.ObjectId, ref: 'User' },
+      teams: [{ type: String }],
+    },
+  ],
+  pendingContributors: [
+    {
+      userId: { type: Schema.Types.ObjectId, ref: 'User' },
+      team: [{ type: String }],
```

Comment:
I'm a little confused here as I am under the impression that each of the pendingContributor could only have one team that they want to work on? If that's the case, I believe the '[{ type: String}]' should be replaced with '{ type: String}' since we don't allow an array of strings. If that's not the case(they could have multiple teams), then maybe changing the attribute name from 'team' to 'teams' might be more in line with the naming of assignmentContributors

Reply:
Nvm I think you are allowed to have multiple teams so I would just suggest changing the attribute name from 'team' to 'teams'

Reply:
Andy you are so right!